DEVELOPING.md

```diff
@@ -44,7 +44,7 @@ The following dependencies must be available to be begin project development.
 
 * [Unix-like][] operating system _— [Windows Subsystem for Linux][] also works_
 * [Git][]
-* [Node][] `^9.7.1`
+* [Node][] `^9.7.1` (don't use a version higher than ~10.0.0 or you may hit errors during setup)
```

Comment:
You will want to move this update into its own `docs:` commit. If you leave it in as is, I believe it might trigger a version bump for every `@hig` package since it is in the root of the repo.